PointFree.playground/Pages/Blog.xcplaygroundpage/Contents.swift

```diff
@@ -7,7 +7,7 @@ import WebKit
 Current = .mock
 let post = Current.blogPosts().first!
 
-let req = request(to: .blog(.show(post)), session: .loggedOut)
+let req = request(to: .blog(.show(.right(post.id))), session: .loggedOut)
```

Comment:
This is the one playground that doesn't seem to run properly. It renders blanks...